scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -33,19 +33,30 @@ oc login -u developer -p password@123 --insecure-skip-tls-verify
 oc whoami
 
 if [ "${ARCH}" == "s390x" ]; then
-    # Integration tests
+    # Integration tests for no cluster
     make test-integration
```

Comment:
Note that `make test-integration` covers both cluster and no-cluster related tests.

Do you want to use `make test-integration-no-cluster`instead? (here and in different other places)

Reply:
No. In our CI no-cluster test cases are failing and test suit exiting without running cluster test cases. We will recheck on this. Thank you